### DISCUSSION TITLE
Gazebo 실행오류

### DISCUSSION THREAD

**Discussion by cheori1** [Road-Balance/gcamp_ros2_basic#23] [Q&A]
Xlaunch 실행 후 위와 같은 명령어 입력후 가제보를 입력하였지만No protocol specified가 계속 떠서 질문 드립니다!
![image](https://user-images.githubusercontent.com/109123326/178399902-6a8d1624-05ac-4283-b9d0-d6a9cfb2706e.png)

**Comment by kimsooyoung** [comment]
xlaunch를 제대로 실행한 후 다시 시도해보세요
